CMSgov/mint-app
# [EASI-3295] Share Model Plans with Users

# EASI-3295

## Changes and Description

- Changes from https://github.com/CMSgov/mint-app/pull/751 and https://github.com/CMSgov/mint-app/pull/758 are included in this feature branch PR
- Ability to Share a model plan with a set of users rather than typing emails directly

## How to test this change

- Navigate to a model plan
- Click the "Share" link
- Share the model plan with some users in the system (by default whoever you're logged in as should be in the DB) as well as some users in the [Okta mock list](https://github.com/CMSgov/mint-app/blob/2dbf6665529c9e4c0df6d46c60ba5957c9d71234/pkg/local/okta_api.go#L1-L492) like `TEST`, `ABCD`, or `GRTB`
- Ensure the email is sent appropriately with MailCatcher
- Ensure users are created in the DB

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
